tests/include/test/drivers/size.h

```diff
@@ -62,16 +62,17 @@
  */
 #define TEST_KEY_CONTEXT_STORE_PUBLIC_KEY   0
 
-/** \def TEST_KEY_CONTEXT_USE_SIZE_FUNCTION
+/** \def TEST_KEY_CONTEXT_SIZE_FUNCTION
  *
- * If this is true a size_function is implemented.
- * If this is false, there is no size_function implemented.
+ * If this is defined a size_function is implemented.
  */
-#define TEST_KEY_CONTEXT_USE_SIZE_FUNCTION  1
+#define TEST_KEY_CONTEXT_SIZE_FUNCTION
```

Comment:
What about to not define it? That way get_expected_key_size() does not always return 0 in the opaque driver case.